vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt

```diff
@@ -321,19 +321,15 @@ class CreateRoomViewModel @AssistedInject constructor(
 //                            preset = CreateRoomPreset.PRESET_PRIVATE_CHAT
 //                        }
 //                    }
-                    // Disabling federation
-                    disableFederation = state.disableFederation
-
```

Comment:
Please can you restore and comment these lines to stay aligned with Element?

Reply:
theses lines are present lines 343. There are not commented, should i do it ?